examples/typescript/pool.ts

```diff
@@ -1,7 +1,8 @@
-import { join } from 'path'
 import { DynamicThreadPool, FixedThreadPool } from 'poolifier'
 import { MyData, MyResponse } from './worker'
 
+import { join } from 'path'
```

Comment:
Please just clean up spaces not needed

Reply:
You mean blank lines? 

The default setup that sort imports put a line separator between pure node.js module import and the other modules. I have to look on how

Reply:
That's ok, we don't need to look into that and complicate the things :)